reference/7.1/Microsoft.PowerShell.Core/About/about_Splatting.md

```diff
@@ -47,6 +47,9 @@ parameters. You may pass some parameters by using splatting and pass others by
 position or by parameter name. Also, you can splat multiple objects in a single
 command so you don't pass more than one value for each parameter.
 
+As of PowerShell 7.1, you can also override a splatted parameter by explicitly
```

Comment:
Is also needed?  Feels like a filler word.

Reply:
good point. Fixed